src/base.jl

```diff
@@ -944,10 +944,6 @@ function make_time_series(
 end
 =#
 
-function get_time_series_names(::Type{T}, component::Component) where {T <: TimeSeriesData}
```

Comment:
This is a useful one to restore in the future.

Reply:
It still exists in IS. If we want it then we need to re-export it in PSY.

Reply:
Yes, please.